### ISSUE TITLE
Fonts not loading properly on Linux

### ISSUE THREAD

**Issue by rytc** [4coder-editor/4coder#205]
Version: 4.1.X - Linux

When first installing the new version of 4coder on Linux, liberation-mono.ttf does not seem to load properly. Font is proportional. After replacing liberation-mono.ttf with a new ttf file, the font loads properly.

However, the font only loads properly if 4coder is launched within its own directory. If 4ed is launched outside its directory, the font does not load and it falls back to the proportional font.

**System**
OS:  ArchLinux

**Comment by aolo2** [comment]
Had the same problem. What did it for me was installing my font of choice to the system directory (/usr/share/fonts/...), instead of 4coder/fonts. I'm not familiar with 4coder font loading code, but it seems that **fonts from system directories are loaded even if they are a worse match than a local ttf**.

This means that if I don't have a font family installed at all, then the usual 4coder/fonts/xxx.ttf loads ok. However, having event the non-monospaced version of the font installed globally overwrites the local monospased ttf (in my case proportional Roboto ttf was being loaded).

That would be my guess.

**Comment by baines** [comment]
Yeah, the code tries to load the font path directly and falls back to passing the name to FontConfig if this fails. There may well be a bug with it though.

**Comment by aolo2** [comment]
The last thing I tried before installing to system folder was actually naming a ttf something completely random (e.g. asd123.ttf) and writing the exact filename to config.4coder. 4coder still failed to load the font, (?) because a similar font was installed system-wide (/?)
